frontend/CHANGELOG.md

```diff
@@ -1,7 +1,10 @@
 # CHANGELOG
 
 ## vNext
-
+- Removed EDP banner
+- Updated sent transfers, received transfers excell format
```

Comment:
It is not only format, you added new columns. Please, update.

Reply:
fixed